resources/charlie/config-example.toml

```diff
@@ -154,6 +154,17 @@ gossip_request_timeout_secs = 10
 get_remainder_timeout_secs = 5
 
 
+# ===================================
+# Configuration options for fetcher
+# ===================================
+[fetcher]
+
+# The timeout duration in seconds for a single fetcher request, i.e. for a single fetcher message
+# sent from this node to another node, it will be considered timed out if the expected response from that peer is
+# not received within this specified duration.
+get_from_peer_timeout = 10
```

Comment:
I think 10 seconds is too much. The minimum round exponent is 4 seconds. I would lower it down to as much as 3~5seconds.

Reply:
changed it to 3s